packages/block-editor/src/components/link-control/test/index.js

```diff
@@ -245,7 +244,8 @@ describe( 'Basic rendering', () => {
 				name: 'Edit',
 			} );
 
-			await user.click( editButton );
+			fireEvent.click( editButton );
```

Comment:
I'm curious about those instances where we actually go back and refactor from `user-library` to `fireEvent`. Could you elaborate on the rationale for that?

Reply:
I'm not sure, but I think it was something like this: `user.click` is an async loop like:
```js
for ( const action of pointerActions ) {
  await wait() // promisified setTimeout( 0 )
  act( () => pointerAction( action ) );
}
```
I.e. there is an async loop and the `act()` calls are synchronous. So, the update from `computePosition().then()` falls in between the `pointerAction` calls and is not wrapped in `act()`.

But it's possible it's all bogus and `user.click` is fine. We'll need to revisit this. It's been a few weeks since I originally did these particular changes and today I know much more about how async `act()` works.